assets/locales/zh-Hant/main.json

```diff
@@ -308,6 +308,7 @@
       "ja": "日文",
       "pl": "波蘭語",
       "pt-br": "葡文 (巴西)",
+      "ru": "俄罗斯",
```

Comment:
How confident are you with these translations in other languages? Do you think it’s necessary to get a native speaker to translate these?

Reply:
To clarify, this is also referring to the other language files that the "Russian" string was added to (Japanese, Arabic, Polish, Finnish, etc.). It's great that you were already thinking about adding these, by the way!